### DISCUSSION TITLE
Accidentally purchased 2 artisan subscriptions

### DISCUSSION THREAD

**Discussion by Morningrevival** [artisan-roaster-scope/artisan#1647] [Q&A]
Artisan told me my first payment method wasn’t processed so I tried a different one and it went through. A few minutes later my first payment method processed and now I have double payed for my subscription. How do I go about getting a refund?

**Comment by poundy** [comment]
You mean Artisan Plus? https://doc.artisan.plus/contact/ has an email address... while Marko is a regular here, and a key developer for Artisan, Artisan Plus is a separately managed entity...
